application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -115,6 +150,81 @@ static boolean handleHasBotPermissions(@NotNull String actionVerb,
         return true;
     }
 
+    private static void handleAbsentTarget(@NotNull String actionVerb, @NotNull Interaction event) {
+        event
+            .reply("I can not %s the given user since they are not part of the guild anymore."
+                .formatted(actionVerb))
+            .setEphemeral(true)
+            .queue();
+    }
+
+    /**
+     * Checks whether the given bot and author have enough permission to change the roles of a given
+     * target. For example whether they have enough permissions to add a role to a user.
+     * <p>
+     * If not, it will handle the situation and respond to the user.
+     * <p>
+     * The checks include:
+     * <ul>
```

Comment:
I would make this an `ol` and write the checks in the order they are checked by the code

Reply:
But then the order becomes a documented and reliable feature that must never be changed again. I dont want that.

Reply:
Then we could also change the docs each time we update this?

Reply:
Thats not really how docs are supposed to work... people should be able to rely on docs. That said, I dont really see a benefit for the user in knowing the order. I only see disadvantages on that. I dont have a strong opinion here though, the effect would be minimal anyways.

Reply:
Yeah it is a very minor change, now that I think of it, I said that only because I could not find anything else to comment upon lol.